diff -up initial-setup-0.3.9.4/initial_setup/product.py.orig initial-setup-0.3.9.4/initial_setup/product.py
--- initial-setup-0.3.9.4/initial_setup/product.py.orig	2013-10-03 13:16:47.000000000 -0400
+++ initial-setup-0.3.9.4/initial_setup/product.py	2013-11-08 13:51:11.188272330 -0500
@@ -6,7 +6,7 @@ import os
 import glob
 
 RELEASE_STRING_FILE = "/etc/os-release"
-LICENSE_FILE_GLOB = "/usr/share/doc/redhat-release*/EULA*"
+LICENSE_FILE_GLOB = "/usr/share/redhat-release*/EULA*"
 
 def product_title():
     """
